### ISSUE TITLE
bad system call

### ISSUE THREAD

**Issue by user122983** [stoeckmann/xwallpaper#35]
info: gentoo, xserver, dwm
everytime i run: "xwallpaper --version" or any other parameters that are valid i get this error: "bad system call"

if anyone thinks he can help please tell me what information should i provide

**Comment by stoeckmann** [comment]
The last lines of "strace xwallpaper --version" can help to track this down. Also, which version of glibc are you using? Reminds me of https://github.com/stoeckmann/xwallpaper/issues/28

**Comment by user122983** [comment]
glibc-2.33

```
seccomp(SECCOMP_SET_MODE_FILTER, SECCOMP_FILTER_FLAG_TSYNC_ESRCH, NULL) = -1 EFAULT (Bad address)
prctl(PR_SET_NO_NEW_PRIVS, 1, 0, 0, 0)  = 0
seccomp(SECCOMP_SET_MODE_FILTER, 0, {len=116, filter=0xee74c0}) = 0
statx(1, "", AT_STATX_SYNC_AS_STAT|AT_NO_AUTOMOUNT|AT_EMPTY_PATH, STATX_BASIC_STATS,  <unfinished ...>) = ?
+++ killed by SIGSYS +++
Bad system call
```

**Comment by stoeckmann** [comment]
I have included statx into the stage 2 sandbox. Let me know if xwallpaper works now. Otherwise I need the next strace call or maybe you can add the system calls into seccomp.c on your own and let me know which ones were still missing.

And thank you for reporting it by the way!

**Comment by user122983** [comment]
fixed the issues now xwallpaper works fine

https://github.com/stoeckmann/xwallpaper/pull/36

**Comment by stoeckmann** [comment]
Thanks! I will release a new version tomorrow.

**Comment by stoeckmann** [comment]
I have released xwallpaper 0.7.1 which fixes this issue. Thanks again for your report and help in fixing this. I close this issue but feel free to re-open if it shows up again.
